src/tools/chip-cert/Cmd_GenCD.cpp

```diff
@@ -125,6 +126,15 @@ const char * const gCmdOptionHelp =
     "\n"
     "       DAC Origin Product Id in hex.\n"
     "\n"
+    "   -a, --authorized-paa-cert <file>\n"
+    "\n"
+    "       File containing PAA certificate authorized to sign PAI which signs the DAC\n"
+    "       for a product carrying this CD. Maximum 10 authorized PAA certificates can\n"
```

Comment:
Please add a note that it is optional and if present, the chaining must remain true for the lifetime of the CD being usable